cobbler/modules/managers/bind.py

```diff
@@ -161,8 +149,8 @@ def __forward_zones(self):
                 host = re.sub(r'\.%s$' % best_match, '', host)
 
                 # if we are to manage ipmi hosts, add that too
-                if self.settings.bind_manage_ipmi:
-                    if system.power_address != "":
+                if (self.settings.bind_manage_ipmi):
+                    if (system.power_address != ""):
```

Comment:
Simply not needed afaik.

```suggestion
                if self.settings.bind_manage_ipmi:
                    if system.power_address != "":
```